app/models/ems_event/automate.rb

```diff
@@ -2,6 +2,14 @@ class EmsEvent
   module Automate
     extend ActiveSupport::Concern
 
+    def graph_refresh(sync: false)
```

Comment:
👍  yeah, graph_refresh is not yet in the code and `manager_refresh` is. So, good consistency catch. Go for the rename